timbermill-client/src/test/java/com/datorama/timbermill/TimberLogAdvancedTest.java

```diff
@@ -1,46 +1,37 @@
 package com.datorama.timbermill;
 
 import com.datorama.timbermill.annotation.TimberLog;
-import com.datorama.timbermill.pipe.LocalOutputPipe;
-import com.datorama.timbermill.pipe.LocalOutputPipeConfig;
 import com.datorama.timbermill.unit.Event;
 import com.datorama.timbermill.unit.LogParams;
 import com.datorama.timbermill.unit.Task;
-import org.awaitility.Awaitility;
 import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Test;
 
-import java.util.concurrent.TimeUnit;
+import java.util.UUID;
```

Comment:
lets extract all orphan tests to a separate class and add 2 additional tests of orphan use case:
7 events that has multiple missing events, A->B->C->D->E->F->G ( B,D,F are missing)
need to check various scenarios